app/helpers/puppetclasses_helper.rb

```diff
@@ -11,8 +11,12 @@ def overridden?(puppetclass)
   end
 
   def puppetclass_group_with_icon(list, selected)
-    (list.last - selected).empty? ?
-        link_to_function(icon_text('plus', list.first, {:class => 'hide'}), "expandClassList($(this), '#pc_#{list.first}')") :
-        link_to_function(icon_text('plus', list.first), "expandClassList($(this), '#pc_#{list.first}')")
+    if (list.last - selected).empty?
+      link_to_function(icon_text('plus', list.first, {:class => 'hide'}),
```

Comment:
I see now there's some duplication in this method, perhaps you could fix it at the same time?  Probably just create the inner hash using the conditional, or the class name(s) to pass in.